src/lib/components/Map/Map.js

```diff
@@ -3,7 +3,7 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Actions, addPopUp, sortLayers, addChart, buildDetailView, prepareLayer } from 'gisida';
-import { detectIE, buildLayersObj, orderLayers } from '../../utils';
+import { detectIE, buildLayersObj, orderLayers, getSharedLayersFromURL } from '../../utils';
```

Comment:
I think we should name `detectIE` to `isIE` or `usesIE` and get rid of the unnecessary assignment `const isIE = detectIE();`